nvflare/private/fed/server/site_security.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+from typing import List, Tuple
+
+from nvflare.apis.event_type import EventType
+from nvflare.apis.fl_constant import FLContextKey
+from nvflare.fuel.hci.conn import Connection
+from nvflare.fuel.hci.proto import InternalCommands
+from nvflare.fuel.hci.server.constants import ConnProps
+from nvflare.fuel.hci.server.reg import CommandFilter
+
+logger = logging.getLogger(__name__)
+
+
+class SiteSecurityFilter(CommandFilter):
```

Comment:
it would be good if we can add some docstring to this class.

Reply:
Added